draco/analysis/flagging.py

```diff
@@ -1361,8 +1459,8 @@ def process(self, tstream, rfimask):
         return tsc
 
 
-class MaskFreq(task.SingleTask):
-    """Apply a mask to the frequency axis.
+class FreqMask(task.SingleTask):
```

Comment:
I'm not so convinced about this rename. I like it when task names describe an action, and this one is now pretty unclear (i.e could it be a container with a frequency mask in it?). The old name wasn't great, but I think this is going backwards slightly.